src/modules/powerrename/lib/Settings.cpp

```diff
@@ -470,12 +359,113 @@ void CRenameMRU::_FreeMRUList()
     }
 }
 
+CSettings::CSettings()
+{
+    std::wstring result = PTSettingsHelper::get_module_save_folder_location(L"PowerRename");
+    jsonFilePath = result + L"\\" + std::wstring(c_powerRenameDataFilePath);
+}
+
+void CSettings::LoadPowerRenameData()
+{
+    if (!std::filesystem::exists(jsonFilePath))
+    {
+        MigrateSettingsFromRegistry();
+
+        SavePowerRenameData();
+    }
+    else
+    {
+        ParseJsonSettings();
+    }
+}
+
+void CSettings::SavePowerRenameData() const
+{
+    json::JsonObject jsonData;
+
+    jsonData.SetNamedValue(c_enabled,                 json::value(settings.enabled));

```

Comment:
As per previous comment, remove this line since we don't save `enabled` in the json.

Reply:
Removed.